### PR TITLE
Feature > Request Geocode

### PR DESCRIPTION
**Pull request by sinakarimi**

### CODE REVIEW

**Comment by sinakarimi** [comment]
[[CC] Geocode Library](https://trello.com/c/TnjayuNg/31-cc-geocode-library)

File: config/config.exs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :ex_geocode,
+  api_host: "https://maps.googleapis.com/maps/api/geocode"
```
Comment (craigp) [inline config/config.exs]: I had to edit the `api_host` to remove the path in order to get this to work, ie. `api_host: https://maps.googleapis.com`

**Review by craigp — CHANGES_REQUESTED** [review]
Aside from the config issue I ran into everything else was working fine 👍

**Review by craigp — APPROVED** [review]

**Comment by craigp** [comment]
Hope you don't mind me committing to this, I was just catching up on PR's I needed to review and thought I'd help out.

**Comment by craigp** [comment]
Are you happy to merge this in?

**Pull request merged**